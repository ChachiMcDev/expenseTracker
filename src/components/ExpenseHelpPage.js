import React from 'react';

const ExpenseHelpPage = ()=> (
    <div>this is the expense help page</div>
)

export default ExpenseHelpPage;