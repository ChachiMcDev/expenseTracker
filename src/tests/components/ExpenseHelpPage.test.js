import React from 'react';
import { shallow } from 'enzyme';
import ExpenseHelpPage from '../../components/ExpenseHelpPage';


test('expect help page to be rendered', ()=>{
    const wrapper = shallow(<ExpenseHelpPage /> );
    expect(wrapper).toMatchSnapshot();
});