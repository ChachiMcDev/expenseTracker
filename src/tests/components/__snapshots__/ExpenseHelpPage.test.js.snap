// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect help page to be rendered 1`] = `
<div>
  this is the expense help page
</div>
`;
